test/models/user_test.rb

```diff
@@ -95,4 +95,24 @@ class UserTest < ActiveSupport::TestCase
 
     assert_equal "Roger Federer", user.full_name
   end
+
+  test "it knows if it is a borrower" do
+    tenant = create(:tenant)
+    tenant.user << create(:user)
+    user = tenant.user.first
```

Comment:
I've just gotten into the habit of using result to make things super clear, for longer tests if we change the test data.